### PR TITLE
UWM-RAR: Added the aria attribtues for accessibility compliance

### PR DESCRIPTION
**Pull request by smitahantzios**

### CODE REVIEW

File: docroot/themes/custom/rar/templates/block/block--next-step--full.html.twig [hunk 1]
```diff
@@ -15,7 +15,7 @@
     page_category.color ? 'block-next-step--' ~ page_category.color|clean_class,
   ]
 %}
-<aside{{ attributes }}>
+<aside{{ attributes }} aria-labelledby="Next Steps">
```
Comment (blaquehat) [inline docroot/themes/custom/rar/templates/block/block--next-step--full.html.twig:18]: @smitahantzios, What is the labelledby referring to?

Based on https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-labelledby_attribute, there should be an id that it links to. I do not see it in the file or in a page (example: /mind/stress/social-distancing-self-care).

File: docroot/themes/custom/rar/templates/block/block--quick-read--full.html.twig [hunk 2]
```diff
@@ -19,7 +19,7 @@
 <div{{ attributes }}>
   {{ title_prefix }}
   {{ title_suffix }}
-  <aside{{ content_attributes.addClass(content_classes)}}>
+  <aside{{ content_attributes.addClass(content_classes)}} aria-labelledby="Article Summary">
     <div class="max-w-screen-sm mx-auto">
```
Comment (blaquehat) [inline docroot/themes/custom/rar/templates/block/block--quick-read--full.html.twig:23]: I am seeing the same concern in this file.

**Review by blaquehat — CHANGES_REQUESTED** [review]

File: docroot/themes/custom/rar/templates/block/block--next-step--full.html.twig (hunk 1 above)
Comment (smitahantzios) [reply docroot/themes/custom/rar/templates/block/block--next-step--full.html.twig]: Fixed

File: docroot/themes/custom/rar/templates/block/block--quick-read--full.html.twig (hunk 2 above)
Comment (smitahantzios) [reply docroot/themes/custom/rar/templates/block/block--quick-read--full.html.twig:23]: Fixed

**Review by blaquehat — APPROVED** [review]

**Pull request merged**